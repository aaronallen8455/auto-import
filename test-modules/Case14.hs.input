module Case14 where

main :: IO ()
main = 1 ::: 2
