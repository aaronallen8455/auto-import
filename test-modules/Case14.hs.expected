module Case14 where

import Ops ((:::))

main :: IO ()
main = 1 ::: 2
